cmake/target/dictionary.cmake

```diff
@@ -30,9 +30,20 @@ endfunction(dictionary_add_global_target)
 # - **FULL_DEPENDENCIES:** MOD_DEPS input from CMakeLists.txt
 ####
 function(dictionary_add_deployment_target MODULE TARGET SOURCES DEPENDENCIES FULL_DEPENDENCIES)
+    run_ac_set("${MODULE}" "autocoder/fpp")
```

Comment:
This is necessary for dictionary generation if the dictionary is an output of the deployment module.